sympy/simplify/combsimp.py

```diff
@@ -217,6 +219,232 @@ def explicate(p):
                 denom_gammas.extend(l)
             elif isg is False:
                 denom_others.extend(l)
+        # =========== level >= 2 work: factor absorbtion =========
+        if level >= 2:
```

Comment:
It may be too early to consider the workings of `find_fuzzy` at this point. There are a lot of other algorithms in `rule_gamma` that do not depend much on other modules. Perhaps you could try to understand what they do and how they should be modified when `gamma` is replaced by `factorial`. Some parts (for example, the one based on the formula `gamma(x)*gamma(1-x) = pi/sin(pi*x)`) do not apply to `factorial` and can be removed.